foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/util/PythonGeneratorUtils.java

```diff
@@ -20,6 +20,7 @@
  * across {@link org.technologybrewery.fermenter.mda.generator.Generator} hierarchies.
  */
 public final class PythonGeneratorUtils {
+    public static String rootArtifactId;
```

Comment:
S: This shared state seems like a bad idea, and doesn't seem cohesive with the class goals at all.  I think we're probably missing some parameter passing somewhere.

Reply:
updated to pass in artifactID instead of setting public static variables.